read-historical-data.md

```diff
@@ -152,3 +152,21 @@ Pay special attention to the following:
     > **Note:**
     >
     > You should use `@@` instead of `@` before `tidb_snapshot` because `@@` is used to denote the system variable while `@` is used to denote the user variable.
+
+## Restoring historical versions
+
+Before you restore data from an older version, make sure that Garbage Collection (GC) does not clear the history data while you are working on it. This can be done by setting the `tidb_gc_life_time` variable as the following example shows. Do not forget to set the variable back to the previous value after the restore.
+
+```sql
+SET GLOBAL tidb_gc_life_time="60m";
+```
+
+To restore data from an older version, you can use one of the following methods:
+
+- For simple cases, use `SELECT` after setting the `tidb_snapshot` variable and copy-paste the output, or use `SELECT ... INTO LOCAL OUTFLE` and use `LOAD DATA` to import the data later on.
+
+- Use [Dumpling](/dumpling-overview.md#export-historical-data-snapshot-of-tidb) to export a historical snapshot. Dumpling performs well in exporting larger sets of data.
+
+> **Note:**
+>
+> Increasing the GC life time from the default 10 minutes to half an hour or more will result in additional versions of rows being retained, which might require more disk space. This might also affect the performance of certain operations such as scans when TiDB needs to skip these additional versions of the same rows during data reads.
```

Comment:
Would it be better to move this note to Line 163? Line 163 is right under how to set the GC life time.